runtime/vm/KeyHashTable.c

```diff
@@ -447,7 +446,7 @@ hashPkgTableIDFor(J9VMThread *vmThread, J9ClassLoader *classLoader, J9ROMClass *
 
 	key.tag = (UDATA)romClass | TAG_ROM_CLASS;
 
-	if (isSystemClassLoader && (J9ROMCLASS_IS_UNSAFE(romClass) || (LOAD_LOCATION_UNKNOWN == locationType))) {
+	if (isSystemClassLoader && (LOAD_LOCATION_UNKNOWN == locationType)) {
```

Comment:
This change is not equivalent code.

Reply:
I realize that; it's not clear what better we can do given the 'unsafe' bit is no longer available (and wasn't entirely reliable).

Reply:
I've restored setting the 'unsafe' bit via that path to the tag bit can continue to be set in the same situations.